test/collection_test.js

```diff
@@ -2266,6 +2281,18 @@ describe("Collection", () => {
         return articles
           .resolve(conflict, resolution)
           .then(() => articles.pullChanges(client, syncResult))
+          .then(result => ({
+            ok: result.ok,
+            lastModified: result.lastModified,
+            errors: result.errors,
+            created: result.created,
+            updated: result.updated,
+            deleted: result.deleted,
+            skipped: result.skipped,
+            published: result.published,
+            conflicts: result.conflicts,
+            resolved: result.resolved,
+          }))
```

Comment:
Maybe we should extract this as a utility function, something like `reifySyncResult` ?